package.json

```diff
@@ -14,7 +14,7 @@
   },
   "main": "tsconfig.json",
   "engines": {
-    "node": ">=10.4.0"
+    "node": ">=16.13.2"
```

Comment:
Node 16 is now EOL, may as well bump this to 18 if we're doing a major?

Reply:
This is a good point. I added the "engines" field to indicate the minimum target of the default compilation. Looking better at https://node.green/#ES2022, we can choose between:
1. 16.11.0
2. 18.0.0
I'll update the field to 18.0.0